exdb/models.py

```diff
@@ -136,6 +136,42 @@ def get_url(self, user):
             return reverse('view_experience', args=[self.pk])
         return reverse('edit', args=[self.pk])
 
+    def get_csv_row(self):
+        row = []
+        row.append(self.name)
+        row.append(self.get_status_display())
+        row.append(str(self.author))
+        planners = ''
+        for planner in self.planners.all():
+            planners += str(planner) + " "
+        row.append(planners)
+        recognition = ''
+        for section in self.recognition.all():
+            recognition += section.name + " "
+        row.append(recognition)
+        row.append(self.start_datetime.strftime("%Y-%m-%d %H:%M"))
+        row.append(self.end_datetime.strftime("%Y-%m-%d %H:%M"))
+        row.append(self.type.name)
+        subtypes = ''
+        for subtype in self.subtypes.all():
+            subtypes += subtype.name + " "
```

Comment:
This results in an extra space at the end of the string. Alternately, you could do:

```python
subtypes = ', '.join([st.name for st in self.subtypes.all()])
```